---
title: "Tracking the Distribution of Brucella abortus in Egypt Based on Core Genome SNP Analysis and In Silico MLVA-16"
collection: publications
permalink: /publication/2021-09-13-Tracking-Brucella-Abortus-Distribution
excerpt: 'The importance of software and web services is changing. So it sufficed to state the software used during scientific data processing, when publishing results in journals. This also applies to web services, which can be methods used in scientific work as well as results of this work. Awareness for the evanescence of tools and services, which hinders the reproducibility of published work, increases. Similar to data repositories, which archive datasets and make them citable, a service for software artifacts would enhance sustainability of science. So we present the CiTAR (Citing & Archiving Research) service in this paper, which enables researches to preserve computational environments and make them citable. In contrast to pure data repositories, CiTAR guarantees the executability of archived environments by providing generic runtimes.'
date: 2021-09-13
venue: 'Microorganisms'
paperurl: 'https://www.mdpi.com/2076-2607/9/9/1942'
citation: 'Holzer, K., El-Diasty, M., Wareth, G., Abdel-Hamid, N. H., Hamdy, M. E., Moustafa, S. A., Linde, J, Bartusch, F, ... & Beyer, W. (2021). Tracking the distribution of Brucella abortus in Egypt based on Core genome SNP analysis and in silico MLVA-16. Microorganisms, 9(9), 1942.'
---
Brucellosis, caused by the bacteria of the genus Brucella, is one of the most neglected common zoonotic diseases globally with a public health significance and a high economic loss among the livestock industry worldwide. Since little is known about the distribution of B. abortus in Egypt, a total of 46 B. abortus isolates recovered between 2012â€“2020, plus one animal isolate from 2006, were analyzed by examining the whole core genome single nucleotide polymorphism (cgSNP) in comparison to the in silico multilocus variable number of tandem repeat analysis (MLVA). Both cgSNP analysis and MLVA revealed three clusters and one isolate only was distantly related to the others. One cluster identified a rather widely distributed outbreak strain which is repeatedly occurring for at least 16 years with marginal deviations in cgSNP analysis. The other cluster of isolates represents a rather newly introduced outbreak strain. A separate cluster comprised RB51 vaccine related strains, isolated from aborted material. The comparison with MLVA data sets from public databases reveals one near relative from Argentina to the oldest outbreak strain and a related strain from Spain to a newly introduced outbreak strain in Egypt. The distantly related isolate matches with a strain from Portugal in the MLVA profile. Based on cgSNP analysis the oldest outbreak strain clusters with strains from the UK. Compared to the in silico analysis of MLVA, cgSNP analysis using WGS data provides a much higher resolution of genotypes and, when correlated to the associated epidemiological metadata, cgSNP analysis allows the differentiation of outbreaks by defining different outbreak strains. In this respect, MLVA data are error-prone and can lead to incorrect interpretations of outbreak events.

[Download paper here](https://www.mdpi.com/2076-2607/9/9/1942)